docs/preview/02-Guidelines/setting-arm-outputs-to-azure-devops-variable-group.md

```diff
@@ -0,0 +1,86 @@
+
+This Bicep template will deploy the Application Insights instance and place the `Id` and `InstrumentationKey` in the output parameters. 
+
+### Updating The Variable Group
+Now all we need to do is execute our [script](https://scripting.arcus-azure.net/Features/powershell/azure-devops#setting-arm-outputs-to-azure-devops-variable-group) which will update the Azure DevOps variable group.
+
+From an Azure DevOps Pipeline this can be done like so:
+``` powershell
+Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
+Install-Module -Name Arcus.Scripting.DevOps  -AllowClobber
+
+Set-AzDevOpsArmOutputsToVariableGroup -VariableGroupName 'myVariableGroup'
+```
+
+### Combining It All In A Pipeline
+Now that we have walked through both steps let's take a look on how to combine this all into a pipeline that can be executed in Azure DevOps.
+For this we use YAML and define two tasks, the first will deploy our Application Insights instance and the second will update our variable group.
+
+``` yaml
+- task: AzureResourceGroupDeployment@3
+  displayName: 'Deploy Bicep template'
+  inputs:
+    azureResourceManagerConnection: 'myServiceConnection'
+    subscriptionId: 'mySubscriptionId'
+    resourceGroupName: 'myResourceGroup'
+    location: 'West Europe'
+    csmFile: 'applicationInsights.bicep'
+    csmParametersFile: 'applicationInsights.parameters.json'
+    deploymentOutputs: ArmOutputs
+
+- task: PowerShell@2
+  displayName: 'Update Variable Group'
+  env:
+    system_accesstoken: $(System.AccessToken)
+  inputs:
+    targetType: 'inline'
+    script: |
+      Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
+      Install-Module -Name Arcus.Scripting.DevOps  -AllowClobber
+
+      Set-AzDevOpsArmOutputsToVariableGroup -VariableGroupName 'myVariableGroup' -ArmOutputsEnvironmentVariableName 'ArmOutputs' -UpdateVariablesForCurrentJob
+```
+
+There are a few things worth noting. First of all we define `deploymentOutputs: ArmOutputs` during the `AzureResourceGroupDeployment@3` task. This means that the output parameters we specified in our Bicep template will be placed in a variable called `ArmOutputs`, this is then referenced during the execution of our script with `-ArmOutputsEnvironmentVariableName 'ArmOutputs'`.
+
+Secondly we use `-UpdateVariablesForCurrentJob` as a parameter when calling the script. This means that the output parameters from the Bicep file are also available as pipeline variables in the current running job. While not necessary in our example here, if you need to deploy another Bicep template that needs output parameters from an earlier deployed Bicep template this is the way to do it.
+
+Finally we use `system_accesstoken: $(System.AccessToken)` in the `Powershell@2` task, this is necessary because we need to use the security token used by the running build. 
+
+## Closing Up
+Using this setup we are able to deploy a Bicep template and update an Azure DevOps variable group with the specified output parameters!
```

Comment:
Personally I don't think adding a screenshot adds a lot of value as it will only show some dummy values. But you have a bit more experience in writing these kinds of guides, so if you think it is a good idea to add a screenshot than let me know and I can add it 👍🏻

Reply:
Yes, you're right. I was thinking of some kind of end-result, but it wouldn't have indeed not much added-value here.